collection-scripts/gather_vms_details

```diff
@@ -132,7 +132,7 @@ else
     exit 1
   fi
 
-  mapfile -t PODS < <(oc get pod --all-namespaces -l kubevirt.io=virt-launcher --no-headers -o custom-columns="NS:.metadata.namespace,NAME:.metadata.name,VM:.metadata.annotations.kubevirt\.io/domain" | awk '{print $1 "_" $2 "_" $3}')
+  mapfile -t PODS < <(oc get pod --all-namespaces -l kubevirt.io=virt-launcher --field-selector=status.phase=Running --no-headers -o custom-columns="NS:.metadata.namespace,NAME:.metadata.name,VM:.metadata.annotations.kubevirt\.io/domain" | awk '{print $1 "_" $2 "_" $3}')
```

Comment:
I'm just wondering if `--field-selector=status.phase!=Completed` will not be safer.

Reply:
IIUC `oc exec` will work only on Pod with the status "running". So isn't it ok to filter only the running pods? The gather_vm_info only runs oc adm inspect on virt-launcher pod and vmi other than oc exec commands. And these outputs will be already collected while it runs gather_vms_namespaces.

Reply:
`Running` is definitively the expected status for virt-launcher pods but I'm just wondering if we can potentially have errating cases where the virt-launcher POD is still declared Pending (it's enough to have a single container of the pod still not running) while maybe we can still get useful debug info from the `compute` container.

Reply:
Ack,  yes we will miss those. Changed it to `--field-selector=status.phase!=Succeeded`.